pkg/virt-handler/vm.go

```diff
@@ -1552,6 +1556,17 @@ func isGuestAgentSupported(vmi *v1.VirtualMachineInstance, commands []v1.GuestAg
 
 func calculatePausedCondition(vmi *v1.VirtualMachineInstance, reason api.StateChangeReason) {
 	switch reason {
+	case api.ReasonPausedMigration:
+		log.Log.Object(vmi).V(3).Info("Adding paused condition")
+		now := metav1.NewTime(time.Now())
+		vmi.Status.Conditions = append(vmi.Status.Conditions, v1.VirtualMachineInstanceCondition{
```

Comment:
Could you please use here the conditionManager with the Update method?

Reply:
Only if you insist :) 

Personally, I don't see a reason, since this condition should either be present or not.

Don't you think it can be misleading to hint that this condition can be ever `False` or overwritten by another paused status?